_episodes/l2-01-managing_contributions.md

```diff
@@ -18,6 +19,52 @@ keypoints:
 - PR are often created to solve specific issues.
 ---
 
+## Multiple branches in remotes
```

Comment:
This section can probably be revised (later)

Reply:
Have opened an issue about it